audio/src/main/java/edu/artic/audio/AudioDetailsViewModel.kt

```diff
@@ -38,31 +55,61 @@ class AudioDetailsViewModel @Inject constructor() : BaseViewModel() {
 
         objectObservable
                 .map {
-                    it.title
-                }.bindTo(title)
+                    it.largeImageFullPath.orEmpty()
+                }.bindTo(image)
                 .disposedBy(disposeBag)
 
-
+        // Note there is a 'credits' property on each AudioTranslation. We do not want that here.
         objectObservable
                 .map {
-                    it.largeImageFullPath.orEmpty()
-                }.bindTo(image)
+                    // This is a credit line for the object, _not_ translated on a per-language basis
+                    it.creditLine.orEmpty()
+                }.bindTo(credits)
                 .disposedBy(disposeBag)
 
 
-        objectObservable
+
+
+        // Retrieve a list of all translations we have available for this object
+        val known = objectObservable
                 .map {
-                    it.audioFile?.transcript.orEmpty()
-                }.bindTo(transcript)
+                    it.audioFile?.allTranslations().orEmpty()
+                }.share()
+
+        known.bindTo(availableTranslations).disposedBy(disposeBag)
```

Comment:
separate line by line for method chaining. though we never explicitly made that the code style, lets do that now.

Reply:
Ok, I can do that.